CHANGELOG.md

```diff
@@ -23,6 +23,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
    + Shows the correct JSON format in the `hints` field
  - #3340, Log when the LISTEN channel gets a notification - @steve-chavez
  - #3184, Log full pg version to stderr on connection - @steve-chavez
+ - #3248, Add more config examples to CLI - @taimoorzaeem
+   + Now accepts ``--example-file``, ``--example-db``, and ``--example-env``
+   + Replace ``-e`` and ``--example`` with ``--example-file``
+   + Rename option ``--dump-schema`` to ``--dump-schema-cache``
```

Comment:
Since they're kinda breaking changes, these should go under a separate subtitle (after the `### Fixed` entries), something like:

```markdown
### Changes
 - #3248, Modified some CLI options - @taimoorzaeem
   + Replace ``-e`` and ``--example`` with ``--example-file``
   + Rename option ``--dump-schema`` to ``--dump-schema-cache``
```

Reply:
Is it possible to make these options non-breaking changes?

Reply:
I initially proposed to not break the options in https://github.com/PostgREST/postgrest/pull/3321#pullrequestreview-1990458404, but we agreed to only use the new ones at the end of that review... if we'd like to not break the options, then we could go with that implementation.

Reply:
Since another breaking change is planned on https://github.com/PostgREST/postgrest/issues/3501#issuecomment-2139656953, I think this should be fine.